cmd/runtimetest/main.go

```diff
@@ -191,6 +191,20 @@ func validateSysctls(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateROPaths(spec, *rspec.Spec) error {
+	fmt.Println("validating Readonly Paths")
+	for _, v := range spec.Linux.ReadonlyPaths {
+		fi, err := os.Stat(v)
```

Comment:
I got from spec `readonlyPaths will set the provided paths as readonly inside the container.`